heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -147,6 +147,31 @@
    */
   public static final String TOPOLOGY_COMPONENT_RAMMAP = "topology.component.rammap";
   /**
+   * Is this topology stateful? The format of this flag is boolean
```

Comment:
Instead of "Is this topology stateful?" can we clarify in the comments what happens when this is set to true/false? For example if I set it to false but use `IStatefulComponent`, I assume I can still be in-memory stateful, but the state will not persist upon instance failures, scaling, restarts, etc. since the framework will not manage the state transfer/persistence (even if I configure a non-memory state provider below). Would setting this to false with a persistent state provider configured be the equivalent to specifying this to be true while configuring an in-memory HashMap provider?